README.md

```diff
@@ -1,79 +1,135 @@
 
-#### File Converter
+#### Filter examples
 
-A file conversion job deployed on AWS Batch is used to support multiple output [file formats](#file-formats). This job
-converts ``.zarr`` expression matrices to the desired file format and writes the result to S3.
+Select all full length cells:
+```
+...
+  "filter": {
+    "op": ">=",
+    "value": "full length",
+    "field": "library_preparation_protocol.end_bias"
+  }
+...
+```
 
-#### DynamoDB
+Select all cells from the "Single cell transcriptome analysis of human pancreas" project with at least 3000 genes
+detected:
+```
+...
+  "filter": {
+    "op": "and",
+    "value": [
+      {
+        "op": "=",
+        "value": "Single cell transcriptome analysis of human pancreas",
+        "field": "project.project_core.project_short_name"},
+      {
+        "op": ">=",
+        "value": 3000,
+        "field": "genes_detected"
+      }
+    ]
+  }
+...
+```
 
-DynamoDB tables are used to track the state and progress of a request. The following is a description of the tables:
+The list of available filter names is available at `/v1/filters`. To retrieve more information about a specific filter,
+GET `/v1/filters/<filter>`.
 
-| **Table name** | **Description** |
-|---|---|
-| Cache table | Caches requests by a hash of its input parameters. |
-| State table | Tracks the progress of a request. |
-| Output table | Stores output values of the request (e.g. file format, errors). |
-| Lock table | Manages locks for across distributed nodes. |
+### Fields
 
+Users can specify a list of metadata fields to be exported with an expression matrix. The list of available metadata
+fields is available at `/v1/fields`. More information about a specific field is available at
+`/v1/fields/<field>`.
 
-### File formats
+### Format
 
-The DCP MS enables users to prepare expression matrices in several formats by supplying the `format` parameter in the
-POST request to the `/matrix` endpoint. The following is a list of supported file formats:
+The Matrix Service supports generating matrices in the following 3 formats:
 
-- [.zarr](https://zarr.readthedocs.io/en/stable/) (default)
-- [.loom](http://loompy.org/)
+- [.loom](http://loompy.org/) (default)
 - [.csv](https://en.wikipedia.org/wiki/Comma-separated_values)
 - [.mtx](https://math.nist.gov/MatrixMarket/formats.html)
 
-The API also makes this information available via the `/matrix/formats` endpoint.
+This list is also available at `/v1/formats` with additional information for a specific format available at
+`/v1/formats/<format>`.
+
+### Feature
 
-## Getting Started
+The Matrix Service also supports generating cell by transcript matrices in addition to cell by gene matrices. To select
+the feature type, specify either `gene` (default) or `transcript` in the POST request. The list of available features is
+available at `/v1/features` with additional information for a specific feature available at `/v1/features/<feature>`.
```

Comment:
"Note that some assay types are incompatible with certain feature types. For example, if `transcript` is selected as the feature type, data from 3' assays will not be included in the generated matrix."